changelog/Makefile

```
@@ -0,0 +1,50 @@
+.PHONY: changelog-generate changelog-regenerate
+
+CHANGELOG_VERSION=1.0.0
+CHANGELOG_FOLDER=unreleased
+changelog-generate:
```

Can we simplify the command name?

```suggestion
generate:
```